src/temperature_component.cpp

```diff
@@ -162,7 +168,7 @@ void TemperatureComponent::setData( const string& varName,
         } else if( varName == D_DIFFUSIVITY ) {
             H_ASSERT( data.date == Core::undefinedIndex(), "date not allowed" );
             diff = data.getUnitval(U_CM2_S);
```

Comment:
Variable names are a bit cryptic sometimes. Consider  perhaps diff -> ocean_heat_diff so that the code is easier to understand. (and perhaps S -> clim_Sens).

Reply:
Agree that the parameter names can be confusing, however that might be a to-do to address post Hector v3